freshmaker/pyxis_gql.py

```diff
@@ -25,6 +25,9 @@
 from gql.dsl import DSLQuery, DSLSchema, dsl_gql
 from gql.transport.requests import RequestsHTTPTransport
 from requests_kerberos import OPTIONAL, HTTPKerberosAuth
+from typing import Dict, List
```

Comment:
Same suggestion about type hinting in this module.

Reply:
Updated